pandas/tslib.pyx

```diff
@@ -335,11 +335,26 @@ class Timestamp(_Timestamp):
 
     def round(self, freq):
         """
-        return a new Timestamp rounded to this resolution
+        Round the Timestamp to the specified resolution
+
+        Returns
+        -------
+        a new Timestamp rounded to the given resolution of `freq`
 
         Parameters
         ----------
         freq : a freq string indicating the rounding resolution
+
+        Raises
+        ------
+        ValueError if the freq cannot be converted
+
+        Notes
+        -----
+        Do not call this function via numpy's round function.
+        That function is for numbers, not timestamps, so no
+        attempts will be made to make it compatible with
+        the numpy implementation.
         """
```

Comment:
I am not sure why you need this at all, ``np.ceil`` completely fails on all things like this and it *should*n't work so I don't see a problem. pls remove all of these types of docs